packages/ra-ui-materialui/src/list/SingleFieldList.tsx

```diff
@@ -174,7 +181,11 @@ const Root = styled('div', {
 // useful to prevent click bubbling in a datagrid with rowClick
 const stopPropagation = e => e.stopPropagation();
 
-// Our handleClick does nothing as we wrap the children inside a Link but it is
```

Comment:
We no longer need this: the default child uses the `clickable` prop only when necessary. And when developers pass a custom child, they'll have to set the `clickable` themselves if they want. In that case, this PR is a slight BC break.

Reply:
Unless I'm mistaken, until now developers always **had** to pass a child. So it's a BC for everyone using `<SingleFieldList>` right now unless they have specifically disabled the link.

I agree it's a very minor BC, but it will still impact almost everyone using `<SingleFieldList>` right now. Maybe we should insist on it more in the docs and/or the changelog?

Reply:
You're right. I updated the PR description, and we should include the "Minor BC break" section in the release changelog.